lib/order.rb

```diff
@@ -0,0 +1,23 @@
+require 'pry'
+class Order
```

Comment:
Could use private methods to make sure you are only making `add` and `total` available publicly.